src/setup-tests.ts

```diff
@@ -73,24 +76,27 @@ const importDB = async (config: ArangoDBConfig,deleteDatabaseFirst = false,updat
     await MigrateWithConfig(Direction.Up, config,migrationsPath);
   }
   
-  if(updateFoxxServiceToDB){
+  if(options.updateFoxxServiceToDB){
     await updateFoxxService(config, config.foxx?.mountPoint, config.foxx?.pathToManifest, config.foxx?.pathZipWithBuild)
   }
  }
 
-
  const pullDownTestDataLocally = async (config: ArangoDBConfig): Promise<void> => {
 
-    // Decide whether to execute the windows script or the linux script
-    let scriptExtension = (process.platform == 'win32') ? 'bat' : 'sh';
-    let location = process.cwd();
-    config.scriptsFolderPath = config.scriptsFolderPath === undefined ? "/node_modules/@bcc-code/arango-migrate/src/util_scripts" : config.scriptsFolderPath
-    let bat =  require.resolve(`${location}${config.scriptsFolderPath}/export-test-data.${scriptExtension}`);
-     bat = `${bat} ${config.url} ${config.auth.username} ${config.databaseName} ${config.auth.password} "${config.testDataPath}"`
-    // Execute the bat script
+
+  let command = `arangodump \\
```

Comment:
Update README.md that we only support linux, if you are a windows developer you have to use the WSL